src/baldr/graphreader.cc

```diff
@@ -305,6 +305,8 @@ bool GraphReader::AreEdgesConnected(const GraphId& edge1, const GraphId& edge2)
     } else {
       const GraphTile* tile = GetGraphTile(n1);
       const NodeInfo* ni = tile->node(n1);
+      if (ni->transition_count() == 0)
+        return false;
```

Comment:
this node only exists on this level so we cant transition. which means it was sending `0` for transition index and getting transitions for the first transition structure in the tile. yeah that isnt good